src/encoded/schemas/file_set.json

```diff
@@ -1,5 +1,6 @@
 {
     "title": "File Set",
+    "description": "A FileSet item consists of the file produced from sequencing a library. Note: each library generated requires its own unique associated FileSet.",
```

Comment:
I think "file" in "A FileSet item consists of the file produced..." should be plural.